inst/htmljs/vendor/quadprog.js

```diff
@@ -0,0 +1,600 @@
+/**
+ * JavaScript implementation of quadratic programming solver
+ * Based on the algorithm from quadprog package
+ */
```

Comment:
where did this code come from?
please add a link / explanation

Reply:
I understand that you copied the code from https://github.com/albertosantini/quadprog/tree/master/lib -- please add a link/reference.
"package quadprog" means a JS package, not R package, right? please make sure that is clear?